visits-service/src/test/java/com/petclinic/visits/datalayer/PersistenceTests.java

```diff
@@ -120,5 +122,19 @@ public void updateVisit(){
         Visit foundVisit = repo.findById(savedVisit.getId()).get();
         assertEquals("Updated Description", foundVisit.getDescription());
     }
+
+    @Test
+    public void findVisitByIdNonExistentVisit(){
+        Optional<Visit> v = repo.findById(234);
+
+        assertEquals(Optional.empty(), v);
+    }
+
+    @Test
+    public void findVisitByIdValid(){
```

Comment:
I dont know why this test doesn't pass with the CI/CD pipeline. The project builds fine when I build it on my end.